levels/d1.md

```diff
@@ -12,9 +12,7 @@ You actively make the team & app grow with sustainability in mind
 ### Before writing code
 
 - Presents technical problems with pro's & con's on multiple potential [solutions](https://github.com/moka-care/levels/blob/guidelines.md#probleme-resolution) spanning across multiple Tech PM - Cooldown or Cycle 
```

Comment:
Navigate?
- quand on a des taches à faire sur d'autres parties de l'app, on peut les faires
vs
- on est autonome même en dehors de moka. est ce que dans une autre boite qui est mobile first, je vais y arriver.

"Mostly autonomous on their scope, require light supervision or technical expertise for validation."
compétence interne jusqu'au niveau D, puis compétence externe.